src/lib/highlight.js

```diff
@@ -0,0 +1,22 @@
+import { getPropertyByPath } from './utils';
+import { component } from './suit';
+
+const suit = component('HighLight');
+
+export default function highlight({
+  attribute,
+  highlightedTagName = 'mark',
+  hit,
+} = {}) {
+  const attributeValue =
+    getPropertyByPath(hit, `_highlightResult.${attribute}.value`) || '';
+
+  return attributeValue
+    .replace(
+      new RegExp('<em>', 'g'),
```

Comment:
What about using a regex litteral instead? `/<em>/g`

Reply:
`/<em>/g` doesn't replace all instances.

Reply:
Sorry, I didn't read correctly. Will change that!